tests/functional/api/test_topics.py

```diff
@@ -10,19 +10,25 @@ def test_topics(gl, gitlab_version):
     create_dict = {"name": "my-topic", "description": "My Topic"}
     if gitlab_version.major >= 15:
         create_dict["title"] = "my topic title"
-    topic = gl.topics.create(
-        {"name": "my-topic", "title": "my topic title", "description": "My Topic"}
-    )
+    topic = gl.topics.create(create_dict)
     assert topic.name == "my-topic"
+
     if gitlab_version.major >= 15:
         assert topic.title == "my topic title"
+
     assert gl.topics.list()
 
     topic.description = "My Updated Topic"
     topic.save()
-
     updated_topic = gl.topics.get(topic.id)
     assert updated_topic.description == topic.description
 
-    topic.delete()
+    create_dict = {"name": "my-second-topic", "description": "My Second Topic"}
+    if gitlab_version.major >= 15:
+        create_dict["title"] = "my second topic title"
+    topic2 = gl.topics.create(create_dict)
+    merged_topic = gl.topics.merge(topic.id, topic2.id)
+    assert merged_topic.id == topic2.id
+
+    topic2.delete()
```

Comment:
This looks great already @lmilbaum just note that we don't return a real object on `merge()`, just the server data, so we can't manipulate it like that.

But I think the server returns enough information that we could even create a proper Topic object that we can then manipulate - might be a nicer interface if we refactor that! :)

I know this is just a draft, just thought I'd chime in.

Reply:
Thanks @nejch for you feedback. The latest change I've pushed just passed the tests. Does that looks good to you?

Reply:
Looks great @lmilbaum, thanks a lot for the tests! :tada: